src/elements/content-sidebar/activity-feed/common/activity-message/__tests__/ActivityMessage.test.js

```diff
@@ -136,4 +136,14 @@ describe('elements/content-sidebar/ActivityFeed/common/activity-message', () =>
         expect(wrapper.state('isTranslation')).toBe(false);
         expect(wrapper.state('isLoading')).toBe(false);
     });
+
+    test('should have TruncatableMessage when `shouldTruncate` is passed', () => {
+        const commentText = {
+            tagged_message: 'How u doing @[2030326577:Young Jeezy]?',
+        };
+
+        const wrapper = shallow(<ActivityMessage id="123" {...commentText} shouldTruncate />);
+
+        expect(wrapper.find('TruncatableMessage').length).toEqual(1);
```

Comment:
Minor, but we typically prefer `expect(wrapper.exists('TruncatableMessage')).toBe(true)`